assemblies/enabling-lab-tuning.adoc

```diff
@@ -0,0 +1,240 @@
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of {org-name} Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+endif::[]
+
+[role='_abstract']
+Data scientists can use LAB-tuning in {productname-short} to run an end-to-end workflow for customizing large language models (LLMs). The link:https://arxiv.org/abs/2403.01081[LAB] (Large-scale Alignment for chatBots) method provides a more efficient alternative to traditional fine-tuning by using taxonomy-guided synthetic data generation (SDG) combined with a multi-phase training process. LAB-tuning workflows can be launched directly from the {productname-short} dashboard using the preconfigured InstructLab pipeline, simplifying the tuning process.
+
+LAB-tuning depends on several {productname-short} components working together to support model customization. It uses data science pipelines to run the tuning workflow, KServe to deploy and serve the teacher and judge models, and the Training Operator to run distributed model training across GPU-enabled nodes. LAB-tuning also relies on the model registry to manage model versions, storage connections (such as S3 or OCI) to store pipeline artifacts and model outputs, and GPU hardware profiles to schedule training workloads.
+
+To enable LAB-tuning, an {openshift-cluster} cluster administrator must configure the required infrastructure and platform components by completing the following tasks:
+
+* Install the required Operators
+* Install the required components
+* Configure a storage class that supports dynamic provisioning
+
+A cluster administrator or a {productname-short} administrator must perform additional setup within the {productname-short} dashboard:
+
+* Make LAB-tuning features visible in the dashboard
+* Create a model registry
+* Create a GPU hardware profile
+
+== Requirements for LAB-tuning
+
+* You have an {openshift-platform} cluster with cluster administrator access.
+* Your {openshift-platform} cluster includes at least one node with 4 GPUs (such as NVIDIA A100s) to run distributed LAB-tuning workloads.
+* Your {openshift-platform} cluster has additional GPU-enabled nodes to serve the teacher and judge models based on the selected model size and serving runtime.
+* Your environment meets the prerequisites for installing the required Operators and using the required components, storage, model registry, and GPU hardware profiles.
+
+== Installing the required Operators for LAB-tuning
+
+To enable your data scientists to customize models using LAB-tuning in {productname-short}, a cluster administrator must install the following Operators in {openshift-platform}:
+
+* {productname-long} Operator
+* {org-name} Authorino Operator (Technical Preview)
```

Comment:
According to the Release Notes draft for RHOAI 2.20 available at [RHOAIENG-21561](https://issues.redhat.com/browse/RHOAIENG-21561) now the recommended version is the stable one

```suggestion
* {org-name} Authorino Operator
```